### ISSUE TITLE
[FEATURE REQUEST] Allow wordlist input for extensions (-x) option

### ISSUE THREAD

**Issue by AffineSecurity** [epi052/feroxbuster#251]
**Is your feature request related to a problem? Please describe.**
The extension option (-x) cannot take a file with a list of common extensions and use that list as extensions for directory busting.

**Describe the solution you'd like**
Allow for wordlists to be parsed by feroxbuster so that a large list of file extensions can be tested. 

**Describe alternatives you've considered**
Provide a preset list of extensions that can be used to enumerate multiple file types quickly.

**Comment by epi052** [comment]
Thanks for the request!

This will effectively be implemented when #242 is added. When that feature is complete, you'll be able to do something like 

```
feroxbuster -u https://example.com/WORD.EXT -w extensions.txt:EXT -w words.txt:WORD
```

For now, you can do some bash tricks to get what you're after (linux assumed)

```
feroxbuster -u https://example.com/  -w words.txt -x $(paste -sd , extensions.txt)
```

**Comment by AffineSecurity** [comment]
Okay awesome! Yeah I figured I could do it via bash but I was hoping I could be lazy and not do that :P
